less/src/focus.less

```diff
@@ -1,7 +1,7 @@
 .focused,
 *:focus,
 input:focus + label {
-    .accessibility.no-touch & {
+    .accessibility@{non-mobile} & {
```

Comment:
`.accessibibilityhtml:not(.os-ios):not(.os-android)` isn't the intended selector

Reply:
Should it be `html:not(.os-ios):not(.os-android).accessibibility` ?

Reply:
`@{non-mobile}.accessibility` should work fine